pkg/util/oc/oc.go

```diff
@@ -14,6 +16,18 @@ func ApplyString(t test.TestHelper, ns string, yaml string) {
 		t.Fatalf("Failed to apply manifest: %v;\nYAML: %v", err, yaml)
 	}
 }
+func ApplyTemplate(t test.TestHelper, ns string, yaml string, vars interface{}) {
+	t.T().Helper()
+	template := template.Run(t, yaml, vars)
+	if err := util.KubeApplyContents(ns, template); err != nil {
+		t.Fatalf("Failed to apply manifest: %v;\nYAML: %v", err, template)
+	}
+}
+func DeleteFromTemplate(t test.TestHelper, ns string, yaml string, vars interface{}) {
+	t.T().Helper()
+	template := template.Run(t, yaml, vars)
```

Comment:
let's rename `var` to `data`, since that's what the go `template` package calls this parameter.

Reply:
Done